src/http/invoke-ws/index.js

```diff
@@ -16,7 +16,7 @@ module.exports = function invokeWebSocketEvent (params, callback) {
   }
 
   // Set up request shape
-  let request = requestFormatter({ req, body, connectionId })
+  let request = requestFormatter({ req, body, requestContext })
```

Comment:
Will mention again below, but I'd really like to see all logic pertaining to the construction of request objects to live in `src/http/invoke-$TYPE/**/_req-fmt.js`